docker-compose.yml

```diff
@@ -42,7 +42,19 @@ services:
   flyway:
     container_name: mds_flyway
     image: boxfuse/flyway:5.2.3-alpine
-    command: -url=jdbc:postgresql://postgres/mds -user=mds -password=test -connectRetries=60 -outOfOrder=true migrate
```

Comment:
I changed the way we were creating test database a while back
https://github.com/bcgov/mds/commit/01bbb61ce16722e255c32e1eb1a64afb8e3fa2a1

Didn't update local to exhibit the same behavior. So this fixes that. Also cleaned up the dockerfile a little